.buildconfig.yml

```diff
@@ -371,6 +371,14 @@ projects:
     path: components/lib/crash
     description: 'A generic crash reporter library that can report crashes to multiple services.'
     publish: true
+  lib-crash-sentry:
+    path: components/lib/crash-sentry
+    description: 'A crash reporter for Sentry.io that that uses lib-crash.'
+    publish: true
+  lib-crash-sentry-legacy:
+    path: components/lib/crash-sentry-legacy
+    description: 'A crash reporter for a hosted version of Sentry that that uses lib-crash.'
```

Comment:
This looks good to me. I'm not sure if we should include the version that is supper for the lib as part of the description?

Reply:
Similar to what we have here https://github.com/mozilla-mobile/android-components/pull/11777/files#diff-ac2097c03183d6b134848941c5d565ed9dedf8fa8cb927050d26bb5eca891c18R29, I guess for the other module I think it will be expected that this will have the latest Sentry version that we support.